app/helpers/scripts_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ScriptsHelper
+
+  def normalize_css_str(str)
+    str.to_s.sub(' ', '-')
+  end
+
+  def list_scenes(project_dir)
+    Dir.glob(project_dir + '/scenes/**/**.m[ab]')
+  end
```

Comment:
This is more appropriate as a model method: `Project#scenes`

Reply:
Done.